tutorials/nats-rdb-offload/index.mdx

```diff
@@ -0,0 +1,229 @@
+2. Paste the following Python code in the file
+    ```
+import asyncio
+from nats.aio.client import Client as NATS
+
+async def run_publisher():
+    # Connect to the NATS server
+    nc = NATS()
+
+    await nc.connect("NATS ACCOUNT URL", user_credentials="NATS CREDENTIALS PATH")
+
+    # Publish a message to the subject
+    await nc.publish("foo", b'Hello, World!')
+
+    # Flush to ensure messages are processed
+    await nc.flush()
+
+    # Close the connection
+    await nc.close()
+
+if __name__ == '__main__':
+    asyncio.run(run_publisher())
+    ```
+3. Update the code with your NATS information
+
+This small Python application will publish a "Hello World!" message under the "foo" subject
+
+## Create the subscriber
+
+We start by creating the subscriber application.
+1. Create a new file in your project directory named nats_subscriber_with_mysql.py
+2. Paste the following Python code in the file
+    ```
+import asyncio
+import mysql.connector
+from nats.aio.client import Client as NATS
+
+async def run_subscriber():
+    # Connect to the NATS server
+    nc = NATS()
+
+    await nc.connect("NATS ACCOUNT URL", user_credentials="NATS CREDENTIALS PATH")
+
+    async def message_handler(msg):
+        subject = msg.subject
+        data = msg.data.decode()
+        print(f"Received a message on '{subject}': {data}")
+
+        # Store the message in the MySQL database
+        conn = mysql.connector.connect(
+            host="PUBLIC ENDPOINT",
+            port="PORT",
+            user="NAME",
+            password="PASSWORD",
+            database="nats_messages"
+        )
+        cursor = conn.cursor()
+        cursor.execute('INSERT INTO messages (subject, data) VALUES (%s, %s)', (subject, data))
+        conn.commit()
+        cursor.close()
+        conn.close()
+
+    # Subscribe to a subject
+    await nc.subscribe("foo", cb=message_handler)
+
+    # Keep the subscriber running to receive messages
+    while True:
+        await asyncio.sleep(1)
+
+if __name__ == '__main__':
+    asyncio.run(run_subscriber())
+```
+3. Update the subscriber code with your Scaleway RDB information as well as the ones form your NATS.
+   You will note that they are the same as the publisher's. 
+   It's completely normal as both applications are communicating through the same NATS server.
+
+## Put it all in motion
+
+You have now your architecture ready : 
+- A NATS server to handle the pub/sub messaging pattern
+- A publisher application that will send a message through the NATS server
+- A subscriber application that will retrieve the message from the NATS server and write it on the mySQL database
+- A database that can be read by the rest of your architecture's clients
+
+To see the flow running, you can follow the steps:
+1. Open your terminal window dedicated to the subscriber and run the python file
+```
+python nats_subscriber_with_mysql.py
+```
+2. Open your terminal window dedicated to the publisher and run the python file
```

Comment:
Earlier in the tuto "Terminal" is capitalized, either or is fine but maybe use only one for consistency, unless you were referring to two types of terminals?

Reply:
I would say to leave it lowercase throughout